alerts/alerts.go

```diff
@@ -44,11 +49,22 @@ type (
 		Timestamp time.Time      `json:"timestamp"`
 	}
 
+	WebHookEventDismissAlerts struct {
+		webhooks.Event
```

Comment:
do we need the tight coupling with `webhooks` here? What if this is:

```go
type eventDismissAlerts struct {
    ToDismiss []types.Hash256 `json:"toDismiss"`
}
```

and we can _register_ this event with the webhooks manager like so:
```go
m.hooks.registerEvent("alerts.dismiss", eventDismissAlerts{ToDismiss: ids}
```

the webhooks manager could have an internal queue to ensure registering an event is not a blocking action, and while processing its queue it would wrap the event in a `WebhookEvent` (which has a `createdAt` field which is set when the event was registered).

Reply:
`registerEvent` thus takes an event id and payload